February-11-2015-Updates.md

```diff
@@ -0,0 +1,52 @@
+# On February 11, 2015, we pushed the following changes:
+
+## Fixing the bottlenecks that made FreeCodeCamp.com so slow
+- Our open source community is growing rapidly, and this week our site has slowed to a crawl during peak hours.
+- We've gone through, identified several bottlenecks, and fixed them. If everything works as planned, Free Code Camp should now be faster than ever.
+- A huge thanks to [@berkeleytrue](https://gitter.im/berkeleytrue) for leading this effort, and to the many campers who helped QA these improvements.
+
+## Introducing the new Wiki
+- Our wiki is now on FreeCodeCamp.com itself (as opposed to on our main project's GitHub  Wiki). 
+- Our wiki pops out like our chat and map do. We welcome feedback on this new functionality. Hotkeys, context aware wiki articles, and other features will come soon.
+- Our wiki has real-time search that improves upon what GitHub offered.
+- A big thanks to [@saintpeter](https://gitter.im/saintpeter), [@hallaathrad](https://gitter.im/hallaathrad), and [@awesomeaniruddh](https://gitter.im/awesomeaniruddh), along with many other campers for their hard work on this!
+
+
+## Retiring Camper News
+- Over the past year, Camper News has been a modest success. But so has our subreddit, which allows for commenting, moderation, and many other features. So we are shifting to Reddit for news sharing. 
+- We recommend creating a Reddit account if you don't already have one and [joining our subreddit](https://www.reddit.com/r/freecodecamp). You can now share links with our entire community there, as well as your thoughts and questions.
```

Comment:
Right now the Reddit is set to "self only" mode.  You're switching it to allow links?  This is a good idea, but we're going to need more vigilant moderation if we do so.  I've personally been visiting once every few days or so, but probably not frequently enough.  We may want to identify an active camper or two there and promote them to mod.